official/utils/logs/logger.py

```diff
@@ -162,6 +162,8 @@ def __init__(self, logging_dir):
     self._logging_dir = logging_dir
     if not tf.gfile.IsDirectory(self._logging_dir):
       tf.gfile.MakeDirs(self._logging_dir)
+    self._metric_file_fh = tf.gfile.GFile(
```

Comment:
nit: "fh" is going to be hard to interpret unless you know what is happening. _metric_file_handle, maybe?